### ISSUE TITLE
[Split Button] Can not using keyboard to focus to split button

### ISSUE THREAD

**Issue by QuangThuanDinh** [microsoft/fluentui#21865]
### Environment Information

- **Package version(s)**: 8.56.1
- **Browser and OS versions**: all browser

### Describe the issue:

![image](https://user-images.githubusercontent.com/77305182/155624158-18b49ef0-646c-4aba-8830-bb1bf4f2fcd7.png)

1. User with keyboadr and screen reader does not know how to using Alt + Arrow Down button to open the Split button menu

2. Another issue with FastPass: Nested interactive controls are not announced by screen readers

#### Expected behavior:

User with keyboard only and screen reader will have a message to tell them how can open the menu flyout.

### Documentation describing expected behavior
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by rwedoff** [comment]
I am running into a similar issue. I cannot adopt the Split button because of this now. Are there any updates on this?

**Comment by sanjeewajayasinghe** [comment]
Having the same issue.  Any update?

**Comment by tomi-msft** [comment]
Hi @QuangThuanDinh, thanks for filing the issue. Can you provide a codepen that repros what you are referring to so I can investigate further? Thanks!

**Comment by RashmiBal** [comment]
You can get a repro of the issue when you run your sample in full page view like this: https://codepen.io/RV122/full/QWmqPXd then run the A11y Insights fast pass tool

![image](https://user-images.githubusercontent.com/46879265/181589941-67d1b828-27c3-4348-a4e4-b276f2b23573.png)

**Comment by RashmiBal** [comment]
> Hi @QuangThuanDinh, thanks for filing the issue. Can you provide a codepen that repros what you are referring to so I can investigate further? Thanks!

You can get a repro of the issue when you run your sample in full page view like this: https://codepen.io/RV122/full/QWmqPXd then run the A11y Insights fast pass tool

**Comment by rwedoff** [comment]
This is a tooling error more than a control error. You can indeed interact with the keyboard, I believe it is alt+ down arrow.
While this is accessible, it won't stop the tooling from flagging this one.

**Comment by khmakoto** [comment]
Yup, like @rwedoff says, this is a tooling error more than a practical one. We aren't likely to change this behavior in v8, but our v9 `SplitButton` control has been rebuilt to better handle keyboard interaction and will not run into this tooling error.

You can take a look at our v9 `SplitButton` control [here](https://react.fluentui.dev/?path=/docs/components-button-splitbutton--default).
